src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage`2.cs

```diff
@@ -95,17 +95,17 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
                 Workspace.AdviseSolutionEvents(solution);
             }
 
-            LoadComponentsInUIContextOnceSolutionFullyLoaded(cancellationToken);
+            await LoadComponentsInUIContextOnceSolutionFullyLoadedAsync(cancellationToken);
```

Comment:
:memo: I suspect the problem is here. Try changing this to use `_ =` instead of `await` (fire and forget)